src/cmake/thirdparty/SetupGeosxThirdParty.cmake

```diff
@@ -314,21 +314,18 @@ if(DEFINED ADIAK_DIR)
                  PATHS ${ADIAK_DIR}
                  NO_DEFAULT_PATH)
 
-    extract_version_from_header( name adiak 
-                                 HEADER "${ADIAK_DIR}/include/adiak.h"
-                                 MAJOR_VERSION_STRING "ADIAK_VERSION"
-                                 MINOR_VERSION_STRING "ADIAK_MINOR_VERSION"
-                                 SUBMINOR_VERSION_STRING "ADIAK_POINT_VERSION")
+    # Header file provides incorrect version 0.3.0
+    message( " ----> adiak_VERSION = 0.2.2")
```

Comment:
programmatic detection is definitely preferable but I think getting adiak updated sooner rather than later is more important

Reply:
> programmatic detection is definitely preferable but I think getting adiak updated sooner rather than later is more important

Opened an issue here in the Adiak repo: https://github.com/LLNL/Adiak/issues/15

Reply:
We could have been using the `thirdPartyLibs/tpl.cpp` file to programmatically gather all the versions.